imagersite/imagersite/settings.py

```diff
@@ -53,11 +57,17 @@
 )
 
 ROOT_URLCONF = 'imagersite.urls'
+LOGIN_URL = 'auth_login'
+LOGIN_REDIRECT_URL = '/profile/'
 
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': [
+            'templates',
+            'imager_images',
+            'imager_profile',
```

Comment:
You don't need to list your applications here, they will be picked up by APP_DIRS below.  And paths here should be absolute paths to the 'templates' directory itself.  The paths will be the starting point for any template lookups.